Documentation/network/servicemesh/ingress.rst

```diff
@@ -58,39 +58,50 @@ Supported Ingress Annotations
 #############################
 
 .. list-table::
-   :widths: 25 25 50
```

Comment:
Ok even looking [at the docs](https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table) I can't find a way to make it change anything. Let's remove.

Reply:
Thanks!